frontend/src/components/landing/Plans.tsx

```diff
@@ -30,64 +31,160 @@ export const Plans = (props: Props) => {
     label: "newlanding-plans-button",
   });
 
+  /** List of premium features **/
+  const premiumFeatures = (
+    <ul className={styles.features}>
+      <li>{l10n.getString("landing-pricing-premium-feature-1-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-3-2")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-4")}</li>
+      <li>{l10n.getString("landing-pricing-premium-feature-5")}</li>
+    </ul>
+  );
+
+  /** List of free features **/
+  const freeFeatures = (
+    <ul className={styles.features}>
+      <li>{l10n.getString("landing-pricing-free-feature-1-2")}</li>
+      <li>{l10n.getString("landing-pricing-free-feature-2")}</li>
+    </ul>
+  );
+
+  /** If premium is not available, show waitlist panel **/
+  const unavailablePremiumPanel = (
+    <a
+      href="/premium/waitlist"
+      className={`${styles.plan} ${styles["wide-plan"]}`}
+    >
+      <img
+        src={RelayPremiumWordmark.src}
+        alt="Firefox Relay Premium"
+        className={styles["word-mark"]}
+      />
+      {premiumFeatures}
+      <div className={styles["plan-description"]}>
+        {l10n.getString("landing-pricing-waitlist-description")}
+      </div>
+      <div ref={premiumFauxButtonRef} className={styles["faux-button"]}>
```

Comment:
This `ref` is used to send a ping to Google Analytics that the user saw a button to purchase Premium, so that we can measure conversions (see `useGaViewPing` above). But since in this case it's impossible to convert, I think we don't want to count this as a view of an upgrade button.

I was thinking that maybe we want a replacement event, but since the waitlist is on our website and we already track page views on our website, I guess we don't need that, so simply removing the `ref` should be fine:

```suggestion
      <div className={styles["faux-button"]}>
```